lib/index.js

```diff
@@ -400,7 +404,7 @@ api.verify = async ({
   } else if(type === 'totp') {
     // test the totpToken provided to the API against the secret drawn from
     // the accounts database
-    verified = totp.verify({token: totpToken, secret: token.secret});
+    verified = authenticator.verify({token: challenge, secret: token.secret});
```

Comment:
Why was this changed from `totp` to `authenticator`? It's unclear to me what this implies, but the documentation does say to change it to suite your use case.  https://www.npmjs.com/package/otplib#in-nodejs

Reply:
It wasn't working with 'totp', but did with 'authenticator'.  Perhaps the defaults are different between the two?